metricbeat/docs/modules/mongodb.asciidoc

```diff
@@ -42,6 +42,27 @@ Or more complex like:
   hosts: ["mongodb://myuser:mypass@localhost:40001", "otherhost:40001"]
 ----------------------------------------------------------------------
 
+some more supported URLS are:
+
+[source,yaml]
+----------------------------------------------------------------------
+- module: mongodb
+  hosts: ["mongodb://localhost:27017,localhost:27022,localhost:27023"]
+----------------------------------------------------------------------
+
+[source,yaml]
+----------------------------------------------------------------------
+- module: mongodb
+  hosts: ["mongodb://localhost:27017/?directConnection=true"]
+----------------------------------------------------------------------
+
+[source,yaml]
+----------------------------------------------------------------------
+- module: mongodb
+  hosts: ["mongodb://localhost:27017,localhost:27022,localhost:27023/?replicaSet=dbrs"]
```

Comment:
We should also handle: **Replica set that [enforces access control](https://www.mongodb.com/docs/manual/tutorial/enable-authentication/), include user credentials**
> mongodb://myDBReader:D1fficultP%40ssw0rd@mongodb0.example.com:27017,mongodb1.example.com:27017,mongodb2.example.com:27017/?authSource=admin&replicaSet=myRepl

Reply:
Applies to all with enforced access control. Are we considering it for all URIs?

Reply:
Ref: https://www.mongodb.com/docs/manual/reference/connection-string/#connection-string-uri-format

Reply:
Also, can you update the docs with this format:
```
mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]
```

Currently, the format mentioned there is:
```
[mongodb://][user:pass@]host[:port]
```

which could be misleading in some cases.

Reply:
Please refer to this section of the doc: https://github.com/elastic/beats/blob/ff1802b978fd4f3c0faa85b4aac1c364e2538cc5/metricbeat/module/mongodb/_meta/docs.asciidoc#module-specific-configuration-notes

The current mentioned format in the doc is: [mongodb://][user:pass@]host[:port][?options], which is valid for simple connection strings and in the same section examples for complex connections are also mentioned, I don't see any confusion there. 

Also, along with the mention of authentication credentials[user:pass@] ,the [?options] part of URI in the specified format, already covers parameters for controlling authentication and access control.

If you still think we need to mention everything explicitly, please let me know.